napari_animation/animation.py

```diff
@@ -243,3 +246,10 @@ def _on_active_keyframe_changed(self, event):
         if active_keyframe:
             keyframe_index = self.key_frames.index(active_keyframe)
             self.set_key_frame_index(keyframe_index)
+
+    def _repr_html_(self):
+        if self._filename is None:
+            return 'Video animation not yet available (use the "animate" method to generate it).'
+        else:
+            html = f'<video width="100%" height="100%" controls> <source src="{self._filename}"> </video>'
```

Comment:
I know it seems silly to have both height and width set to 100%, but this actually seems to generate nice responsive and resize-able video windows as output, without me having to guess whether the video is likely to be portait or landscape (and also whether the notebook screen space is portrait or landscape).